oio/event/filters/webhook.py

```diff
@@ -19,11 +19,15 @@
     oio_exception_from_httperror
 from oio.common.json import json
 from oio.event.evob import Event, EventError
+from oio.event.consumer import EventTypes
 from oio.event.filters.base import Filter
+from oio.container.client import ContainerClient
 
 
 class WebhookFilter(Filter):
+
     def init(self):
+        self.container_client = ContainerClient(self.conf, logger=self.logger)
```

Comment:
This lets me think that we must review how we create connection pools in the event-agent. We would better share a big one with a known configuration instead of instantiating several with default configuration.